src/components/tab/Tabs.tsx

```diff
@@ -0,0 +1,96 @@
+import { Children, FC, PropsWithChildren, ReactElement, useMemo, useState } from 'react';
+import { TabProps } from './Tab';
+import classNames from 'classnames';
+import { max } from '@popperjs/core/lib/utils/math';
+
+export type TabTypes = 'default' | 'underline' | 'pills' | 'fullWidth';
+export type ItemStyleTypes = 'active' | 'notActive';
+
+export const listStyleClasses: Record<TabTypes, string> = {
+  default: 'flex-wrap border-b border-gray-200 dark:border-gray-700',
+  underline: 'flex-wrap -mb-px',
+  pills: 'flex-wrap font-medium text-sm text-gray-500 dark:text-gray-400',
+  fullWidth:
+    'hidden text-sm font-medium hidden rounded-lg divide-x divide-gray-200 shadow sm:flex dark:divide-gray-700 dark:text-gray-400',
+};
+export const itemStyleClasses: Record<TabTypes, Record<ItemStyleTypes, string>> = {
+  default: {
+    active: 'bg-gray-100 text-blue-600 dark:bg-gray-800 dark:text-blue-500',
+    notActive:
+      'text-gray-500 hover:bg-gray-50 hover:text-gray-600 dark:text-gray-400 dark:hover:bg-gray-800  dark:hover:text-gray-300',
+  },
+  underline: {
+    active: 'text-blue-600 rounded-t-lg border-b-2 border-blue-600 active dark:text-blue-500 dark:border-blue-500',
+    notActive:
+      'border-b-2 border-transparent text-gray-500 hover:border-gray-300 hover:text-gray-600 dark:text-gray-400 dark:hover:text-gray-300',
+  },
+  pills: {
+    active: 'rounded-lg bg-blue-600 text-white',
+    notActive: 'rounded-lg hover:text-gray-900 hover:bg-gray-100 dark:hover:bg-gray-800 dark:hover:text-white',
+  },
+  fullWidth: {
+    active:
+      'inline-block p-4 w-full text-gray-900 bg-gray-100 focus:ring-4 focus:ring-blue-300 active focus:outline-none dark:bg-gray-700 dark:text-white',
+    notActive:
+      'bg-white hover:text-gray-700 hover:bg-gray-50 focus:ring-4 focus:ring-blue-300 focus:outline-none dark:hover:text-white dark:bg-gray-800 dark:hover:bg-gray-700',
+  },
+};
+
+export type TabsProps = PropsWithChildren<{
+  style?: TabTypes;
+}>;
+
+export const Tabs: FC<TabsProps> = ({ children, style = 'default' }) => {
```

Comment:
It would be better to have the same approach as the other components, like `Tab.Group` and `Tab.Item`